### ISSUE TITLE
Some CBA Settings fail to import

### ISSUE THREAD

**Issue by tricona3** [CBATeam/CBA_A3#1620]
**Mods (complete and add to the following information):**
- **Arma 3:** `2.14`
- **CBA:** `CBA_A3 v3.16.0.231017`
- Various other mods (ACE3, TFAR Beta, CUP Weapons/Units/Vehicles, multiple other mods including private ones.
_Make sure to reproduce the issue with only CBA on a newly created mission!_

**Description:**
Some CBA settings now fail to import, giving the error in the RPT that they don't exist, despite them definitely existing. This wasn't a problem until this CBA update, and now it is doing it across various mods, including ones that haven't updated in years (so I'm ruling out them being the issue)

The issue is happening when either importing from a saved preset via the load button, or when pasting in the settings via the import button.

If I put the settings into a cba_settings.sqf file in the mission directory it does work however. But even if I then try to save that as a preset it then won't work when importing it to a fresh mission.

**Steps to reproduce:**
- Make a new mission in Eden
- Attempt to import CBA Settings from a saved preset
- Does not import some settings, gives error in RPT that they don't exist

**Expected behavior:**
I would expect all settings to import as they did previously

**Where did the issue occur?**
- Editor (Singleplayer)

**Log Files:**
- [RPT file](https://pastebin.com/aWdT0MWY)
- [The preset I am attempting to import](https://pastebin.com/1QkVrqDa)

**Additional context:**
It's really strange how it seems to be random settings that are doing this, some of these mods haven't updated in a long time so there's no reason they should have stopped working. As mentioned previously it was fine before the CBA update.

**Screenshots:**
![example1](https://github.com/CBATeam/CBA_A3/assets/108828234/52b305fa-099e-41f1-ada9-8d8460dd7306)
The screenshot just shows one of the example settings, just to prove that it definitely does exist. This is taken after attempting to import the settings, you can see it hasn't forced the setting despite that being in the preset.

**Comment by johnb432** [comment]
This reason why it's throwing the errors is because it considers the category comments as part of the first setting's name.
E.g. this is considered to be a setting name:
```
Setting // ACE Advanced Ballistics
force ace_advanced_ballistics_ammoTemperatureEnabled
```

The `cba_settings.sqf` file is preprocessed, which means all of the comments are deleted before it's processed. However, when importing, this does not happen and so the comments are part of the processing - something that is not intended.

A temporary fix is to remove the comments (e.g. `// ACE Advanced Ballistics`) from the file and then import the preset.

**Comment by tricona3** [comment]
Ahh, I see. I wonder if this is due to the change in this update for parsing multiple lines?

Thanks for the temporary fix though. Will do that for the time being

**Comment by jonpas** [comment]
> I wonder if this is due to the change in this update for parsing multiple lines?
Likely yes.

**Comment by jonpas** [comment]
Fixed by #1622.
